.env.example

```
@@ -0,0 +1,103 @@
+# =============================================================================
+# Creative TV – Environment variables template
+# Copy to .env.local and fill in values. On Vercel, set these in Project Settings.
+# =============================================================================
+
+# --- Core (required by config/index.ts) ---
+NEXT_PUBLIC_ALCHEMY_API_KEY=
+NEXT_PUBLIC_SUPABASE_URL=
+NEXT_PUBLIC_SUPABASE_ANON_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)